sample_apps/simple_app_under_oe/oe_policy.json

```diff
@@ -1,7 +1,6 @@
 {
-  "policyKey" : "policy_key_file.bin",
-
-  "measurements" : [
-    "c7037df41e773056809af62acf78a107d8e43a0277fbc3431e5539bc8e1ee12a"
+  "policyKey": "policy_key_file.bin",
```

Comment:
Should this file be updated every time? I think this diff should be withdrawn, but need input from the reviewers.